src/Table/Table.tsx

```diff
@@ -104,10 +104,15 @@ const GraaspTable: React.FC<Props> = ({
   rowDragManaged = true,
   enableDrag = false,
 }) => {
+  const [rowData, setRowData] = useState(initialData);
   const [gridApi, setGridApi] = useState<GridApi>();
   const [selected, setSelected] = useState<string[]>([]);
   const classes = useStyles({ tableHeight });
 
+  useEffect(() => {
+    setRowData(initialData);
+  }, [initialData]);
+
```

Comment:
Is this effect required if you already set the initial `rowData`? Or does this force a re-render?

Reply:
Yeah we need to force a re-render, because if you stay on the same component but the prop changed (in my memberships, I delete a membership from the parent, it should propagate but) then it doesn't update.